docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -0,0 +1,74 @@
+# YAML getting started - YAML triggers (not yet available, for discussion only)
+
+## Schema
+
+```yaml
+ci:           # Simple syntax is an array of include-branches. We had previously discussed
+- master      # "branches" but in this proposal "ci" is used more heavily:
+- releases/*  # 1) "ci: false"
+              # 2) full ci syntax (batch, branch include/exclude, path include/exclude
+              # 3) on each artifact type
+              #
+              # So for consistency, "ci" might make more sense. Note, for TFVC "branches"
+              # is not accurate terminology.
+```
+
+```yaml
+ci: none # Turn off CI.
+```
+
+```yaml
+ci:
+  batch: bool | number
+  branches:
+    include: []
+    exclude: []
+  paths:
+    include: []
+    exclude: []
+
+schedules:
+  whenUnchanged: bool
```

Comment:
Schedules are an array of individual schedules.

```yaml
schedules:
- schedule
   whenUnchanged: bool
    days:
    time: # we could probably take the time and time zone info as a singe string hh:mmTZD
    timeZone:
    branches:
```

Reply:
oops i thought i had it that way